### DISCUSSION TITLE
Implement Todo.txt format

### DISCUSSION THREAD

**Discussion by madeindjs** [GitJournal/GitJournal#803] [Feature Requests]
I know that GitJournal handle only Markdown files but it may be great if app can handle [Todo.txt](http://todotxt.org/). I use [Simpletask](https://f-droid.org/app/nl.mpcjanssen.simpletask) which is near perfect and open source.

**Comment by vHanda** [comment]
GitJournal also handles `.txt` files now! (Still not shipped).

I'm hesitant to add Todo.txt format support as that would take this project away from note taking. The interfaces required for Todo tracking are very different than note taking. Maybe it would be simpler to just ask SimpleTask to add git support?

**Comment by mmoya** [comment]
vHanda, first of all, the application is GREAT. It's young and thus has its rough edges, but it just works.

Regardind this, adding git support to SimpleTask would be a solution but if #44 and #93 are implemented, GitJournal will become a 100% (or near) GTD/todotxt compatible ToDo tracker.

**Comment by vHanda** [comment]
Hey Meykil. Thank you.

Could you please also file issues for any/all of the rough edges that you notice? I've been working on this app long enough that I surely now have a lot of blind spots.
